django/cantusdb_project/main_app/management/commands/sync_source_creation_date.py

```diff
@@ -0,0 +1,87 @@
+from django.core.management.base import BaseCommand
+from main_app.models import (
+    Source,
+)
+import requests, json
+import datetime
+from django.utils import timezone
+
+
+def get_source_list():
+    source_list = []
+    file = open("source_list.txt", "r")
+    for line in file:
+        line = line.strip("\n")
+        source_list.append(line)
+    file.close()
+    return source_list
+
+
+def convert_epoch_to_date_time(epoch):
+    try:
+        epoch_time = float(epoch)
+        datetime_str = datetime.datetime.fromtimestamp(epoch_time)
+        datetime_obj = timezone.make_aware(datetime_str, timezone.utc)
+    except (ValueError, TypeError) as e:
+        print(f"Error converting epoch time: {e}")
+        return None
+    return datetime_obj
+
+
+def update_date_created(source_id):
+    url = f"http://cantus.uwaterloo.ca/json-node/{source_id}"
+    response = requests.get(url)
+    json_response = json.loads(response.content)
+    try:
+        date_created_epoch = json_response["created"]
+    except (KeyError, TypeError) as e:
```

Comment:
At this point, if you've checked that "created" is not in `json_response` then you don't need the `try/except ...`.